packages/progress-bar/src/ProgressBar/hooks/useScreenReaderAnnouncer.tsx

```diff
@@ -41,7 +41,7 @@ export function useScreenReaderAnnouncer({
       }
     }
 
-    if (newThresholdIndex === thresholdIndexRef.current) return;
+    if (newThresholdIndex === thresholdIndexRef.current) return '';
```

Comment:
style nit: even when there's only one line after the `if`, we tend to prefer wrapping it in a block `{ return '' }`